docs/JLBP-14.md

```diff
@@ -11,18 +12,16 @@ same code with the same compiler at different
 times can produce different artifacts with different behavior.
 Version ranges can pull in incomplete releases that break the
 build. For example, a [version range in appengine-gcs-client's
-dependency on google-http-java-client broke multiple customers' 
+dependency on google-http-java-client broke multiple customers'
 builds](https://github.com/GoogleCloudPlatform/appengine-gcs-client/issues/71)
-when only half the artifacts in the project were pushed one afternoon. 
+when only half the artifacts in the project were pushed one afternoon.
 
-Version ranges can be a security hole. When a new version of a dependency
-is pushed to Maven Central, it is picked up by downstream dependencies
```

Comment:
"library with a version range" implies that the library you depend on is using a version range, which isn't what you're going for. I think you want working like "A project that uses a version range when depending on a library picks up new versions..."

Reply:
done